cmd/ko/resolve.go

```diff
@@ -87,7 +87,7 @@ func makePublisher(no *NameOptions, lo *LocalOptions) (publish.Interface, error)
 
 		repoName := os.Getenv("KO_DOCKER_REPO")
 		if lo.Local || repoName == publish.LocalDomain {
-			return publish.NewDaemon(namer), nil
+			return publish.NewDaemon(namer, []string{}), nil
```

Comment:
Shouldn't we also pass through the tags options for a local `resolve`?

Reply:
yup, add this here and on the `makePublisher`